### PR TITLE
pep8 cleanup

### PR DESCRIPTION
**Pull request by wtgee**
95% of this is just applying pep8 to the files.  Some additional removal of old code.  A few minor fixes.

### CODE REVIEW

File: huntsman/camera/__init__.py [hunk 1]
```diff
@@ -152,10 +152,8 @@ def create_distributed_cameras(camera_info, logger=None):
     for cam_name, cam_uri in camera_uris.items():
         logger.debug('Creating camera: {}'.format(cam_name))
         cam = PyroCamera(port=cam_name, uri=cam_uri)
-        is_primary = ''
         if primary_id == cam.uid or primary_id == cam.name:
             cam.is_primary = True
-            is_primary = ' [Primary]'
```
Comment (wtgee) [inline huntsman/camera/__init__.py:158]: @AnthonyHorton did you intend to alter the camera string property? This was unused.

File: huntsman/tests/test_camera.py [hunk 2]
```diff
@@ -233,7 +235,7 @@ def test_exposure_scaling(camera, tmpdir):
     else:
         fits_path = str(tmpdir.join('test_exposure_scaling.fits'))
         camera.take_exposure(filename=fits_path, dark=True, blocking=True)
-        image_data, image_header = fits.getdata(fits_path, header=True)
+        image_data, image_header = fits_utils.getdata(fits_path, header=True)
```
Comment (wtgee) [inline huntsman/tests/test_camera.py:238]: This was an error. Not sure how test was passing.

File: states/huntsman/dithering.py [hunk 3]
```diff
@@ -25,4 +25,4 @@ def on_enter(event_data):
             pocs.next_state = 'observing'
 
     except Exception as e:
-        pocs.logger.warning("Problem with preparing: {}".format())
+        pocs.logger.warning("Problem with preparing: {}".format(e))
```
Comment (wtgee) [inline states/huntsman/dithering.py:28]: This exception would have been raising an exeption.

File: huntsman/tests/test_camera.py (hunk 2 above)
Comment (AnthonyHorton) [reply huntsman/tests/test_camera.py:238]: This tests usually gets skipped as it depends on a camera having a bit depth other than 16 bits/pixel and the huntsman-pocs test suite only tests with a single type of simulated camera (Pyro, SDK, 16 bit, with focuser). Running the tests on a variety of simulated cameras like in POCS is on my todo list.

File: huntsman/camera/__init__.py (hunk 1 above)
Comment (AnthonyHorton) [reply huntsman/camera/__init__.py:158]: ISTR there being an `is_primary` variable being actually  used in this area of code at some point. I think these lines are orphaned code fragments from times past... Deleting them is the merciful thing to do.

Comment (wtgee) [reply huntsman/camera/__init__.py:158]: This exists in POCS when the camera is created:

https://github.com/panoptes/POCS/blob/5dbd821b582eedac992da9124defcacc03b0e09a/pocs/camera/__init__.py#L183-L190

**Review by AnthonyHorton — APPROVED** [review]
LGTM. Might want to throw in this `is` -> `=` while you're doing it, though: https://github.com/AstroHuntsman/huntsman-pocs/blob/591e16b12766dfa6c178be5dfdcbaf0639b26b44/states/huntsman/calibrating.py#L26

**Pull request merged**